src/app/_components/KeyMemberCard.tsx

```diff
@@ -1,34 +1,41 @@
+import Image from 'next/image'
+
 import { LinkedinLogo } from '@phosphor-icons/react/dist/ssr'
 
 import type { MemberData } from '@/types/memberType'
 
+import type { Speaker } from '@/schemas/event/SpeakerSchema'
+
 import { Card } from '@/components/Card'
 import { Heading } from '@/components/Heading'
-import { StaticImage } from '@/components/StaticImage'
 
-type KeyMemberCardProps = MemberData
+type KeyMemberCardProps = MemberData | Speaker
+
+export function KeyMemberCard(props: KeyMemberCardProps) {
+  const { name, title, linkedin, image } = props
+  const isStaticImage = typeof image.src === 'object'
 
-export function KeyMemberCard({
-  name,
-  title,
-  linkedin,
-  image,
-}: KeyMemberCardProps) {
   return (
     <li className="relative flex rounded-lg border border-brand-500 p-1">
-      <StaticImage
-        data={image}
+      <Image
+        src={image.src}
         alt={`Photo of ${name}`}
         sizes="150px"
         className="aspect-[3/4] w-32 rounded object-cover"
+        {...(isStaticImage
+          ? { placeholder: 'blur' }
+          : { width: 150, height: 200 })}
       />
 
       <div className="m-3 grow">
         <Heading tag="h3" variant="lg">
           {name}
         </Heading>
 
-        <p className="mt-1 text-brand-300">{title}</p>
+        <p className="mt-1 text-brand-300">
+          {title}
+          {'company' in props && props.company && `, ${props.company}`}
```

Comment:
I might be missing something - why the check? isn't company required?

Reply:
For `Speakers` it is, but for for `MemberData` (used for Filecoin Foundation Board members, for example, on the company page, it isn't.